components/example-notebook-servers/jupyter-pytorch-gaudi/Dockerfile

```diff
@@ -0,0 +1,143 @@
+#
+# NOTE: Use the Makefiles to build this image correctly.
+#
+
+ARG BASE_IMG=<jupyter>
+FROM $BASE_IMG
+
+# Content below is extracted from scripts/Dockerfiles here:
+# https://github.com/HabanaAI/Setup_and_Install/tree/main/dockerfiles/base
+# https://github.com/HabanaAI/Setup_and_Install/tree/main/dockerfiles/pytorch
+
+# version details
+ARG ARTIFACTORY_URL=vault.habana.ai
+ARG VERSION=1.16.2
+ARG REVISION=2
+ARG PYTORCH_VERSION=2.2.2
+ARG OS_NUMBER=2204
+
+# runtime variables
+ENV DEBIAN_FRONTEND=noninteractive
+ENV GC_KERNEL_PATH=/usr/lib/habanalabs/libtpc_kernels.so
+ENV HABANA_LOGS=/var/log/habana_logs/
+ENV HABANA_SCAL_BIN_PATH=/opt/habanalabs/engines_fw
+ENV HABANA_PLUGINS_LIB_PATH=/opt/habanalabs/habana_plugins
+ENV TCMALLOC_LARGE_ALLOC_REPORT_THRESHOLD=7516192768
+ENV PYTHONPATH=/home/$NB_UID:/usr/lib/habanalabs/
+
+# There is no need to store pip installation files inside docker image
+ENV PIP_NO_CACHE_DIR=on
+ENV PIP_DISABLE_PIP_VERSION_CHECK=1
+
+# libfabric and scaling variables
+ENV LIBFABRIC_VERSION="1.20.0"
```

Comment:
Is it possible to factor this into an `ARG` rather than an `ENV`, as we do for all other versions?